rust/private/rust.bzl

```diff
@@ -521,18 +521,31 @@ def _tidy(doc_string):
     Returns:
         str: A string optimized for stardoc rendering
     """
-    return "\n".join([line.strip() for line in doc_string.splitlines()])
+    lines = doc_string.splitlines()
+    if not lines:
+        return doc_string
+
+    # Detect how much space prepends the first line and subtract that from all lines
+    space_count = len(lines[0]) - len(lines[0].lstrip())
+
+    # If there are no leading spaces, do not alter the docstring
+    if space_count == 0:
+        return doc_string
+    else:
+        # Remove the leading block of spaces from the current line
+        block = " " * space_count
+        return "\n".join([line.replace(block, "", 1).rstrip() for line in lines])
 
 _common_attrs = {
     "aliases": attr.label_keyed_string_dict(
-        doc = _tidy("""
+        doc = _tidy("""\
```

Comment:
Why not skip the first line if empty in `_tidy`and not require \? It seems fragile to remember to put \ there.

Reply:
Added support for ignoring first lines.